pkg/utils/tekton/pipelines.go

```diff
@@ -87,6 +89,23 @@ type VerifyEnterpriseContract struct {
 }
 
 func (p VerifyEnterpriseContract) Generate() *v1beta1.PipelineRun {
+	var snapshot app.SnapshotSpec
+	err := json.Unmarshal([]byte(p.Image), &snapshot)
+	if err != nil {
+		fmt.Printf("Application Snapshot doesn't exsit: %s\n", err)
```

Comment:
typo: exsit

Reply:
Good eye @rsoaresd, thanks.

Reply:
@rsoaresd seems that this was addressed can you revise your review?

Reply:
@rsoaresd  addressed typo, could you help to have another look to help merge?